.github/workflows/deploy.yml

```
@@ -3,45 +3,109 @@ on:
+          cd ${DST_PATH}/frontend &&
+          echo 'NEXT_PUBLIC_API_URL=${{ secrets.NEXT_PUBLIC_API_URL }}' > .env.local"
         env:
           SSH_USER: ${{ secrets.SSH_USER }}
           SSH_HOST: ${{ secrets.SSH_HOST }}
           DST_PATH: ${{ secrets.DST_PATH }}
 
-          
+      # サービスの再起動
+      - name: Restart services
+        run: >
+          ssh -i ${{ runner.temp }}/key -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${SSH_USER}@${SSH_HOST} 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)